### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Mathieu Emery

### DISCUSSION THREAD

**Discussion by mathieuemery** [heig-vd-dai-course/heig-vd-dai-course#232] [Show and tell]
https://github.com/mathieuemery/mathieuemery

**Comment by ludelafo** [comment]
Nice profile!

It seems one of the picture in your profile does not show correctly as its source does not exist (the link in the `src` attribute is invalid).

Thanks for making me discover <https://github.com/DenverCoder1/readme-typing-svg>. :)
